zqd/api/api.go

```diff
@@ -109,7 +109,6 @@ type PcapPostStatus struct {
 
 type LogPostRequest struct {
 	Paths          []string             `json:"paths"`
-	StopErr        bool                 `json:"stop_err"`
```

Comment:
Removing this option from the api can happen in separate PR.

Reply:
ok